administrator/components/com_associations/Documentation.md

```diff
@@ -0,0 +1,38 @@
+#### Introduction
+
+to do.
+
+#### Associations Component (com_associations) Requirements
+
+1. Component should have a `ROOT/components/<com_componentname>/helpers/association.php` Example [here](https://github.com/joomla/joomla-cms/blob/staging/components/com_content/helpers/association.php)
+
+2. `ROOT/component/<component_name>/helpers/association.php` needs to load `route.php`:
+`JLoader::register(<ComponentName>'HelperRoute', JPATH_SITE . '/components/<com_componentname>/helpers/route.php');` Example [here](https://github.com/joomla/joomla-cms/blob/staging/components/com_content/helpers/association.php#L13)
+
+3. Component using Categories with associations should have the method `getCategoryAssociations` in `ROOT/component/<component_name>/helpers/association.php`. Example [here](https://github.com/joomla/joomla-cms/blob/staging/components/com_content/helpers/association.php#L33)
+
+4. Component with associated items should have `$associationsContext` and `$typeAlias` defined in the model concerned. Example [here](https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_content/models/article.php#L35) and [here](https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_content/models/article.php#L43)
+
+5. Component should have in back-end specific `modal_associations.php` views for the item, parallel to the edit view. Example [here](https://github.com/joomla/joomla-cms/tree/staging/administrator/components/com_content/views/article/tmpl) Another way is to have chosen fields, any other implementation will not work.
```

Comment:
backend, not back-end
why the link is not direct to the file like the other links?